doc/man7/openssl_user_macros.pod.in

```diff
@@ -40,6 +53,30 @@ meaningful:
 
 =back
 
+For convenience, higher numbers are accepted as well, as long as
+feasible.  For example, C<0x60000000L> will work as expected.
+However, it is recommended to start using the second form instead:
+
+=item C<m>
+
+This form is a simple number that represents the major version number
+and is supported for version 3.0.0 and up.  For extra conventience,
+these numbers are also available:
```

Comment:
I mean the change in the definition of `OPENSSL_API_COMPAT` specifically. We could choose to handle this separately to the `OPENSSL_VERSION` macros if we wanted. I guess my main concern is the ensure that applications that define the old OPENSSL_API_COMPAT format will still work - which is sort of related to the question I had about whether all the headers include opensslconf.h

Reply:
Is this code `opensslconf.h.in` good enough an answer?

https://github.com/openssl/openssl/blob/1ef17a85285155f09cbb85e92b68c341db196534/include/openssl/opensslconf.h.in#L100-L103

This is designed to work with the old form for *quite* some time, so people will have time (years, I suspect!) to change their habit to use just the major number instead of the hex code.

Reply:
Ok